cypress/support/api_commands.js

```diff
@@ -238,7 +243,7 @@ Cypress.Commands.add('apiSaveThemePreference', (value = JSON.stringify(theme.def
 Cypress.Commands.add('loginAsNewUser', (user = {}) => {
     const timestamp = Date.now();
 
-    const {email = `newE2ETestUser${timestamp}@mattermost.com`, username = `NewE2ETestUser${timestamp}`, password = 'password123'} = user;
+    const {email = `newe2etestuser${timestamp}@sample.mattermost.com`, username = `NewE2ETestUser${timestamp}`, password = 'password123'} = user;
```

Comment:
Update the email to lowercase (since where saving it in server that way), so we don't need to worry in doing lowercase when asserting this value.